src/assets/levels.json

```diff
@@ -1,11 +1,35 @@
 [
   {
     "type": "money",
+    "goal": 10010
```

Comment:
You're not wrong, but that's why I think we should increase the difference between purchase and sales price. Simply make it easier to earn money!